docs/api/core.md

```diff
@@ -5,79 +5,89 @@
 
 .. autoclass:: AECEnv
```

Comment:
Why, in the right hand side navbar, is AECEnv in code format? The navbar format is also still different from https://gymnasium.farama.org/api/core/#

Reply:
This is what was meant  by " AEC Env split attributes and functions on the content table to the right. See gymnasium core"

Reply:
> Why, in the right hand side navbar, is AECEnv in code format?
Because they were generated using .. autoclass:: , ..py:function and etc. 

>  The navbar format is also still different from

I'll change the headings then